pages/index.tsx

```diff
@@ -24,18 +24,14 @@ export default function Home(): JSX.Element {
       <Main>
         <Section>
           <TextBlock>
-            <h2 id="a-gathering">
-              <a className="return" href="#">
```

Comment:
This triggered a [JSX A11y error for invalid `href`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md) so removed for now.

Reply:
Using `/` will result in a full page reload which is a bit different to the intended scroll to top. We could tweak the config for this rule if we wanted I guess. But yeah might be something we can create a story to look into.

Reply:
I think that's why I used `#` (I'm sure I tried `/` first). Annoying. Could add an `id="top"` up there somewhere and scroll to the top, but that leaves us with an anchor at the top of the page. But we'll be redoing this soon so to hell with it all. I was hoping for a nice way to do this that would work even on older browsers, but it's not that long a page so screw it.